libs/jsonschema/schema.go

```diff
@@ -40,6 +40,10 @@ type Schema struct {
 
 	// Default value for the property / object
 	Default any `json:"default,omitempty"`
+
+	// If specified this is the list of valid values for the object. Values
+	// here should be consistent with the type defined in the schema.
+	Enum []any `json:"enum,omitempty"`
```

Comment:
Please add validation for this.

Reply:
I have added validation for config values. Something that's missing is validating whether enum values and default values are valid during schema load. I think that's fine to skip for now. I have created an internal ticket to track this.

Reply:
Let's not skip this.

We already call `schema.validate()` on load and should include a check for the enum values there. We have to be robust to user errors. Who knows what happens downstream if we pass on an invalid schema.